sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/AvroIO.java

```diff
@@ -326,6 +327,14 @@
       }
 
       @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        builder.add("schema", type);
+        if (filepattern != null) {
+          builder.add("filePattern", filepattern);
+        }
```

Comment:
how do you want to handle the properties for nested transforms?

Specifically, filepattern is really just passed directly through to `FileBasedSource` -- should it be there instead?